src/app/features/home/components/news/news-page/news-page.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { NewsItem } from '../../../../../core/models/news';
+import { Observable } from 'rxjs';
+import { NewsService } from '../../../services/news.service';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-flat-news-page',
+  templateUrl: './news-page.component.html',
+  styleUrls: ['./news-page.component.scss']
+})
+export class NewsPageComponent implements OnInit {
+
+  newsCard?: Observable<NewsItem|undefined>;
```

Comment:
remove `?`

Reply:
if I remove ? it shows me mistake " Property 'newsCard' has no initializer and is not definitely assigned in the constructor."

Reply:
yes, that's why you should fix your eslint config